contracts/crowdsale/FinalizableCrowdsale.sol

```diff
@@ -28,12 +30,11 @@ contract FinalizableCrowdsale is Crowdsale, Ownable {
     isFinalized = true;
   }
 
-  // end token minting on finalization
-  // override this with custom logic if needed
+  /**
+   * @dev Can be overriden to add finalization logic. The overriding function
+   * should call super.finalization() to ensure the chain of finalization is
+   * executed entirely.
+   */
   function finalization() internal {
-    token.finishMinting();
   }
```

Comment:
I thought about it. If it's left abstract you can't call `super.finalization()` in the deriving contract, which we want everyone to do to make sure the whole `finalization` chain runs.